app/services/fdc_service.rb

```diff
@@ -0,0 +1,6 @@
+class FdcService
+  def self.search(query)
+    response = Faraday.get("https://api.nal.usda.gov/fdc/v1/foods/search?query=#{query}&pageSize=10&api_key=#{ENV['FDC_API_KEY']}")
```

Comment:
I would set my parameters of `query, api_key, and pageSize` in a block to increase the readability of this request. You could also extract both the json parsing and a  Faraday connection object into their own methods to DRY up this class. Granted, for this challenge, you weren't asked to make multiple requests. But, it's good practice to get in that habit because the later projects and the final will likely ask you to make multiple requests to the same API service.